sites/kit.svelte.dev/scripts/types/index.js

```diff
@@ -65,7 +69,14 @@ async function get_types(code, statements) {
 					// @ts-ignore
 					const jsDoc = statement.jsDoc[0];
 
-					comment = jsDoc.comment;
+					// resolve `@link` JSDoc tags
+					if (Array.isArray(jsDoc.comment)) {
```

Comment:
When is this an array of comments? When there's one or more `@` directives in there, like `@link` or `@param` etc? Does this in any way affect other directives besides `@link`?